spec/services/recaptcha_enterprise_validator_spec.rb

```diff
@@ -0,0 +1,305 @@
+require 'rails_helper'
+
+describe RecaptchaEnterpriseValidator do
+  let(:score_threshold) { 0.2 }
+  let(:analytics) { FakeAnalytics.new }
+  let(:extra_analytics_properties) { {} }
+  let(:action) { 'example_action' }
+  let(:recaptcha_enterprise_api_key) { 'recaptcha_enterprise_api_key' }
+  let(:recaptcha_enterprise_project_id) { 'project_id' }
+  let(:recaptcha_site_key_v3) { 'recaptcha_site_key_v3' }
+  let(:assessment_url) do
+    "#{described_class::BASE_VERIFICATION_ENDPOINT}/#{recaptcha_enterprise_project_id}" \
+      "/assessments?key=#{recaptcha_enterprise_api_key}"
```

Comment:
same as before, I'm not a huge fan of the `" \` string concatenation style

Reply:
Updated in 80f00242d.